api-channel-jira/src/main/java/com/synopsys/integration/alert/api/channel/jira/distribution/custom/MessageReplacementValues.java

```diff
@@ -40,14 +42,16 @@ public MessageReplacementValues(
         @Nullable String projectVersionName,
         @Nullable String componentName,
         @Nullable String componentVersionName,
-        @Nullable String severity
+        @Nullable String severity,
+        @Nullable String policyCategory
     ) {
```

Comment:
It doesn't need to happen now, but this class has enough nullable fields that it should probably have a builder.

Reply:
That may be a good idea for a future improvement for this class. Not including this change, we have a request for 4 additional custom fields in 6.7.0